vnext/Desktop.IntegrationTests/WebSocketJSExecutorIntegrationTest.cpp

```diff
@@ -51,6 +51,7 @@ TEST_CLASS (WebSocketJSExecutorIntegrationTest) {
   }
 
   BEGIN_TEST_METHOD_ATTRIBUTE(LoadApplicationScriptSucceeds)
+  TEST_IGNORE()
```

Comment:
>TEST_IGNORE [](start = 2, length = 11)

@acoates-ms believed we weren't providing enough to actually load a bundle and disabled this.